frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView.stories.tsx

```diff
@@ -252,7 +252,7 @@ function ScrollDecorator(Story: Story) {
 
 function DarkBackgroundDecorator(Story: Story) {
   return (
-    <Box bg="bg-dark" mih="100vh">
+    <Box bg="#434e56" mih="100vh">
```

Comment:
I changed the background color because I just realize the footer border has the same color as the background so it's invisible.